src/slider/slider.component.ts

```diff
@@ -72,6 +72,14 @@ import { EventService } from "carbon-components-angular/utils";
 						(mousedown)="onMouseDown($event)"
 						(keydown)="onKeyDown($event)">
 					</div>
+					<div
+						#thumb2
```

Comment:
Just a heads up, the slider in the future should add more aria-* attributes to be properly accessible.

Reply:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-1.html

Reply:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-2.html